content/terminal/custom-backend/Widgets/table/table.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Table Example
+sidebar_position: 1
+description: Learn how to integrate your own backend with OpenBB Terminal Pro using the cookie-cutter or language-agnostic API approaches, with illustrative guides and principles for handling widget.json files, APIs, interfaces, Python, FastAPI, and more.
+keywords:
+- widgets.json
+- OpenBB API
+- Endpoint integration
+- widget configuration
+- Language-Agnostic API
+- API implementation
+- Python
+- FastAPI
+- Terminal Pro widgets
+- Widget definitions
+---
+
+# Introduction
+
+This guide will walk you through the process of creating a table widget for OpenBB Terminal Pro. By the end of this guide, you will have a working table widget that you can add to OpenBB Terminal Pro.
+
+> **Prerequisite:** Ensure you have Python > 3.11 installed on your system before proceeding.
+
+## Step 1: Set Up Your Project
+
+To get started, create the main application file and the widget configuration file. You will only need these two files:
+
+- `main.py`: This file will contain your FastAPI application code.
+- `widgets.json`: This file will define the configuration for your widget.
+
+## Step 2: Create the Backend
+
+The backend will use the same FastAPI setup and structure as described in the [Custom Backend](/content/terminal/custom-backend/custom-backend.md#create-your-own-backend) page.
+
+### Edit the Main Application File
```

Comment:
Same as in the chart tutorial:
Edit the Main Application File -> Create the table endpoint